src/BtTreeModel.cpp

```diff
@@ -841,12 +844,6 @@ void BtTreeModel::deleteSelected(QModelIndexList victims)
          case BtTreeItem::MISC:
             Database::instance().remove( misc(ndx) );
             break;
-         case BtTreeItem::RECIPE:
-            Database::instance().remove( recipe(ndx) );
-            break;
-         case BtTreeItem::STYLE:
-            Database::instance().remove( style(ndx) );
-            break;
```

Comment:
I have exactly 0 clues what happened there. As soon as I figure out how to "rebase and repush", I will fix this.